src/main/java/org/openrewrite/staticanalysis/RemoveUnreachableCodeVisitor.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.staticanalysis;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.java.JavaVisitor;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.Statement;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class RemoveUnreachableCodeVisitor extends JavaVisitor<ExecutionContext> {
```

Comment:
Maybe best to start with this visitor as package private to limit the publicly exposed API. 
```suggestion
class RemoveUnreachableCodeVisitor extends JavaVisitor<ExecutionContext> {
```

Reply:
Good thinking, I'm not used to public library development. 😄